shared/cfg/guest-os/Linux/Fedora/31.cfg

```diff
@@ -0,0 +1,96 @@
+        # Unattended-file does not require any changes
+        unattended_file = unattended/Fedora-25.ks
+        unattended_install.url:
+            # Installation works fine with mem=1024 on methods such as cdrom
+            # but fails ("No space left on device") with methods such as url.
+            mem = 2048
+            url = http://dl.fedoraproject.org/pub/fedora-secondary/releases/31/Server/${vm_arch_name}/os/
+        # ARCH dependent things
+        aarch64:
+            arm64-mmio:
+                # Only latest updates contain virtio driver
+                inactivity_watcher = none
+                take_regular_screendumps = no
+            kernel_params = "console=ttyAMA0 console=ttyS0 serial"
+            unattended_install.cdrom:
+                md5sum_cd1 = 041e59c5b4bc2f05c9d8bdc7b697fef8
+                md5sum_1m_cd1 = 89b0a217e0eef61f0c60360e560199ce
+            unattended_install.url:
+                url = http://dl.fedoraproject.org/pub/fedora/linux/releases/31/Server/${vm_arch_name}/os
+                sha1sum_vmlinuz = 3505f2751e2833c681de78cee8dda1e49cabd2e8
+                sha1sum_initrd = df6acb53fa1ce6e15c2d65bda18a4ac8ea77701a
+        ppc64le:
+            kernel_params = "console=hvc0 serial"
+            boot_path = ppc/ppc64
+            unattended_install.cdrom:
+                md5sum_cd1 = e673856a2547121044ee372d184cd552
+                md5sum_1m_cd1 = 400455f87e58a976696ca25df331887b
+            unattended_install.url:
+                sha1sum_vmlinuz = 11ece09fe5a8ff5838027fe3f653adcfb6f26efe
+                sha1sum_initrd = 2de8c72f683ca875de3620f63c52e53709dbbb54
+        s390x:
+            kernel_params = "console=ttysclp0 serial"
+            boot_path = images
+            kernel = images/f31-s390x/kernel.img
+            # Anaconda hardcodes headless installation of F28 on s390x
+            vga = none
+            inactivity_watcher = none
+            take_regular_screendumps = no
+            unattended_install.cdrom:
+                md5sum_cd1 = ca302d6a5c21d1bfe41b871b75197767
+                md5sum_1m_cd1 = d459f5d89931a29d199e605be03c84a9
+            unattended_install.url:
+                sha1sum_vmlinuz = b93d1efcafcf29c1673a4ce371a1f8b43941cfeb
+                sha1sum_initrd = 3de45d411df5624b8d8ef21cd0b44419ab59b12f
+        x86_64:
+            kernel_params = "console=tty0 console=ttyS0"
+            unattended_install.cdrom:
+                md5sum_cd1 = af29363e2c8825948515aac36701fbfa
+                md5sum_1m_cd1 = 8c09623eee12715472242f844cf69e4e
+            unattended_install.url:
+                url = http://dl.fedoraproject.org/pub/fedora/linux/releases/31/Server/${vm_arch_name}/os
+                sha1sum_vmlinuz = 5b6f6876e1b5bda314f93893271da0d5777b1f3c
+                sha1sum_initrd = 0d7b4bdccb3ca82e0a221104d9f20eda0cc53309
+        # Shared specific setting
+        unattended_install.url:
+            kernel_params += " inst.repo=${url}"
```

Comment:
You removed the warning that this should be removed in the future (note it's already fixed in RHEL but it doesn't seem to be fixed in F31 so it has to be here...)

Reply:
^^ I mean would you please include the note here so we can remove this hopefully in 32 (or 33?)

Reply:
^^sure,I didn't know requesting inst.repo is a bug.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1649359